[2022-02-21 16:53:24,011] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-03T00:00:00+00:00 [queued]>
[2022-02-21 16:53:24,025] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-03T00:00:00+00:00 [queued]>
[2022-02-21 16:53:24,025] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:24,025] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 16:53:24,025] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:24,040] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-02-03 00:00:00+00:00
[2022-02-21 16:53:24,046] {standard_task_runner.py:52} INFO - Started process 125 to run task
[2022-02-21 16:53:24,049] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-02-03T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp6_3luc5l', '--error-file', '/tmp/tmpjt7ne7h0']
[2022-02-21 16:53:24,049] {standard_task_runner.py:77} INFO - Job 37: Subtask get_path
[2022-02-21 16:53:24,101] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-02-03T00:00:00+00:00 [running]> on host 7c24e16e6467
[2022-02-21 16:53:24,149] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-02-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-03T00:00:00+00:00
[2022-02-21 16:53:24,150] {logging_mixin.py:109} INFO - /opt/***
[2022-02-21 16:53:24,150] {python.py:175} INFO - Done. Returned value was: None
[2022-02-21 16:53:24,164] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220203T000000, start_date=20220221T165324, end_date=20220221T165324
[2022-02-21 16:53:24,220] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 16:53:24,248] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 17:02:08,751] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-03T00:00:00+00:00 [queued]>
[2022-02-21 17:02:08,764] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-03T00:00:00+00:00 [queued]>
[2022-02-21 17:02:08,764] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:02:08,764] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 17:02:08,764] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:02:08,779] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-02-03 00:00:00+00:00
[2022-02-21 17:02:08,785] {standard_task_runner.py:52} INFO - Started process 124 to run task
[2022-02-21 17:02:08,788] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-02-03T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpditc5sta', '--error-file', '/tmp/tmprafdld3m']
[2022-02-21 17:02:08,788] {standard_task_runner.py:77} INFO - Job 35: Subtask get_path
[2022-02-21 17:02:08,836] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-02-03T00:00:00+00:00 [running]> on host ce809d68944a
[2022-02-21 17:02:08,891] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-02-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-03T00:00:00+00:00
[2022-02-21 17:02:08,891] {python.py:175} INFO - Done. Returned value was: /nba_database
[2022-02-21 17:02:08,915] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220203T000000, start_date=20220221T170208, end_date=20220221T170208
[2022-02-21 17:02:08,960] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 17:02:09,010] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 19:10:49,616] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-03T00:00:00+00:00 [queued]>
[2022-02-21 19:10:49,623] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-02-03T00:00:00+00:00 [queued]>
[2022-02-21 19:10:49,623] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:49,623] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 19:10:49,623] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:49,630] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-02-03 00:00:00+00:00
[2022-02-21 19:10:49,633] {standard_task_runner.py:52} INFO - Started process 123 to run task
[2022-02-21 19:10:49,635] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-02-03T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpa93rg1wu', '--error-file', '/tmp/tmp77aoi1uy']
[2022-02-21 19:10:49,635] {standard_task_runner.py:77} INFO - Job 35: Subtask get_path
[2022-02-21 19:10:49,668] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-02-03T00:00:00+00:00 [running]> on host 1594888d6c38
[2022-02-21 19:10:49,709] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-02-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-03T00:00:00+00:00
[2022-02-21 19:10:49,709] {python.py:175} INFO - Done. Returned value was: ['/home/***/.local/bin', '/usr/local/lib/python37.zip', '/usr/local/lib/python3.7', '/usr/local/lib/python3.7/lib-dynload', '/home/***/.local/lib/python3.7/site-packages', '/usr/local/lib/python3.7/site-packages', '/opt/***/dags', '/opt/***/config', '/opt/***/plugins']
[2022-02-21 19:10:49,721] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220203T000000, start_date=20220221T191049, end_date=20220221T191049
[2022-02-21 19:10:49,767] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 19:10:49,807] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
